src/Http/Middleware/LogWithRequestContext.php

```diff
@@ -277,8 +281,8 @@ protected function logMultiPart(string $messageContext, string $message): void
 
     protected function getUserDisplay(): string
     {
-        if ($this->maybeUserId == null) {
-            return "unauthenticated user";
+        if ($this->maybeUserId === null) {
```

Comment:
Any chance `$this->maybeUserId` is actually undefined and not null? Adding tertiary equality check would fail that use case.

Reply:
Nope, because I'm explicitly setting it to `null`, or `null` would be returned by the other functions that evaluate for the ID.

Reply:
I'll change it to an `isset` call though

Reply:
nvm, thought this was js for a moment 😬